api/management/commands/acquiredata.py

```diff
@@ -27,3 +27,13 @@ def handle(self, *args, **options):
         if not os.path.exists(sample_path):
             sample_url = 'https://raw.githubusercontent.com/cognoma/cancer-data/master/data/samples.tsv'
             urlretrieve(sample_url, sample_path)
+
+        gene_path = os.path.join(options['path'], 'genes.tsv')
+        if not os.path.exists(gene_path):
+            gene_url = 'http://www.stephenshank.com/genes.tsv'
+            urlretrieve(gene_url, gene_path)
+
+        mutation_path = os.path.join(options['path'], 'mutation-matrix.tsv.bz2')
+        if not os.path.exists(mutation_path):
+            mutation_url = 'https://ndownloader.figshare.com/files/5864862'
```

Comment:
@jessept -- nice. My main worry _here_ is that hardcoding the URL for a specific dataset of a specific version of the figshare data is going to cause an upkeep issue later on. So the goal will be to use the cognoml logic for figshare downloads to avoid repeating any efforts and clean this up!